pyomo/contrib/parmest/examples/reactor_design/updatesuffix_example.py

```diff
@@ -0,0 +1,67 @@
+#  ___________________________________________________________________________
+#
+#  Pyomo: Python Optimization Modeling Objects
+#  Copyright (c) 2008-2025
+#  National Technology and Engineering Solutions of Sandia, LLC
+#  Under the terms of Contract DE-NA0003525 with National Technology and
+#  Engineering Solutions of Sandia, LLC, the U.S. Government retains certain
+#  rights in this software.
+#  This software is distributed under the 3-clause BSD License.
+#  ___________________________________________________________________________
+
+from pyomo.common.dependencies import numpy as np, pandas as pd
+from os.path import join, abspath, dirname
+import pyomo.contrib.parmest.parmest as parmest
+from pyomo.contrib.parmest.examples.reactor_design.reactor_design import (
+    ReactorDesignExperiment,
+)
+
+import pyomo.environ as pyo
+from pyomo.contrib.parmest.utils.model_utils import update_model_from_suffix
+
+
+def main():
+    # Read in file
+    # Read in data
+    file_dirname = dirname(abspath(str(__file__)))
+    file_name = abspath(join(file_dirname, "reactor_data.csv"))
+    data = pd.read_csv(file_name)
```

Comment:
This isn't a deal-breaker, but the inconsistency with how the files are loaded between the two examples makes me squint. In `update_suffix_doe_example`, you do:
```python
    # Read in file
    DATA_DIR = pathlib.Path(__file__).parent
    file_path = DATA_DIR / "result.json"
```

Here you do the more "classic" `dirname`, `abspath`, `join` routine. It just kind of looks weird, if that makes sense?